features/step_definitions/fincap_pages_steps.rb

```diff
@@ -0,0 +1,67 @@
+Given(/^I have an insight page type$/) do
+  english_site = Comfy::Cms::Site.find_or_create_by(
+    label: 'en',
+    identifier: 'money-advice-service-en',
+    hostname: 'localhost:3000',
+    path: 'en',
+    locale: 'en',
+    is_mirrored: true
+  )
+
+  english_site.layouts.find_or_create_by(
+    identifier: 'insight',
+    label: 'Insight',
+    content:  <<-CONTENT
+      {{ cms:page:content:rich_text }}
+      {{ cms:page:overview }}
+      {{ cms:page:countries }}
+      {{ cms:page:links_to_research }}
+      {{ cms:page:contact_details }}
+      {{ cms:page:year_of_publication }}
+    CONTENT
+  )
+end
+
+Given(/^I am on the homepage$/) do
+  home_page.load
+end
+
+Given(/^I click on "([^"]*)"$/) do |button|
+  page.click_on(button)
+end
+
+Given(/^I select "([^"]*)" on the creation page$/) do |option|
+  new_page.select(option)
+end
+
+Given(/^I create a page "([^"]*)"$/) do |title|
+  new_page.page_name.set(title)
+  new_page.save_button.click
+end
+
+When(/^I edit the page "([^"]*)"$/) do |title|
+  home_page.load
+  page.first(:link, title).click
```

Comment:
Is there not a specific identifier for the edit button? There must be a better way of finding a clickable object within the page. This element can be set in the page object class and used just as is in previous parts of this PR i.e. `page.edit_button.click`

What will happen if the title is changed in the slightest?

Reply:
not sure of the best solution to this. On the homepage there are two identical links to edit the page. Neither of the links have any specific identifier and the parents of them a fairly generic as well. The benefit of this solution is that *if* the title in the feature is changed slightly then it won't find the page, which should happen. We want to ensure this step is selecting the correct page to edit and the best way of ensuring this is to be using the title of the page, don't you think?

Reply:
@willhall88 If that's the case then maybe this solution is safe to toe the same line.